Project2015To2017/ProjectConverter.cs

```diff
@@ -24,11 +24,16 @@ public sealed class ProjectConverter
 		private readonly ILogger logger;
 		private readonly ConversionOptions conversionOptions;
 		private readonly ProjectReader projectReader;
+		private readonly ITransformationSet transformationSet;
 
-		public ProjectConverter(ILogger logger, ConversionOptions conversionOptions = null)
+		public ProjectConverter(
+			ILogger logger,
+			ConversionOptions conversionOptions = null,
+			ITransformationSet transformationSet = null)
```

Comment:
Is this parameter really optional? I would just have the implementor specify one and not default to anything but throw instead.

Reply:
That was intended to be required argument, but later I changed it to optional to preserve API contract. And it is used in analyzers (after conversion).